### ISSUE TITLE
Wrong color parsing

### ISSUE THREAD

**Issue by czebe** [alvaropinot/aco-reader#2]
This lib would be very useful.
However when testing with #ffffff and #000000 swatches, the white colour is incorrectly parsed as #0000ff = blue

**Comment by alvaropinot** [comment]
Good catch! This is just a quick PoC but that scenario should be inside the test cases :)

PR's welcomed 😄

**Comment by alvaropinot** [comment]
Just did a quick test case for that and seems like it's working fine. Can you please provide a test .aco file for that scenario? :)

**Comment by czebe** [comment]
Yes, I have a swatch exported from Photoshop with white and black colors only.
Output is #0000ff for the white swatch. Here's the aco file and my basic script:
https://github.com/czebe/swatch-names

**Comment by alvaropinot** [comment]
Taking a look into it now. I'll let you know :D

**Comment by alvaropinot** [comment]
```
❯ xxd fixtures/swatches.aco   
00000000: 0001 0002 0001 0000 0000 ffff 0000 0000  ................
00000010: 0000 0000 0000 0000 0002 0002 0001 0000  ................
00000020: 0000 ffff 0000 0000 0006 0057 0068 0069  ...........W.h.i
00000030: 0074 0065 0000 0000 0000 0000 0000 0000  .t.e............
00000040: 0000 0006 0042 006c 0061 0063 006b 0000  .....B.l.a.c.k..
```

Taking a quick look to your swatches.aco, seems like it's actually blue 😅 even if the name of the color is White as you can see in bytes 20..30. In the first line you can see the 00000000: 0001 0002 0001 **0000 0000 ffff** 0000 0000. If I use a third party tool I also get a blue "white" hehehe. Is photoshop actually showing a white white?

<img width="762" alt="screen shot 2017-10-16 at 20 26 32" src="https://user-images.githubusercontent.com/6654199/31623459-8cd81480-b2b0-11e7-8428-0236e496418c.png">

The swatch I've created for testing looks like this, please note that is inverted Black and then White.

```
❯ xxd fixtures/test-black-and-white.aco 
00000000: 0001 0002 0000 0000 0000 0000 0000 0000  ................
00000010: ffff ffff ffff 0000 0002 0002 0000 0000  ................
00000020: 0000 0000 0000 0000 0006 0042 006c 0061  ...........B.l.a
00000030: 0063 006b 0000 0000 ffff ffff ffff 0000  .c.k............
00000040: 0000 0006 0077 0068 0069 0074 0065 0000  .....w.h.i.t.e..
```
here is the correct ff's -> 00000030: 0063 006b 0000 0000 **ffff ffff ffff** 0000  .c.k............

Could you please create a new swatch?

**Comment by alvaropinot** [comment]
Hmmm, seems like it could be the colour space as your's is `0001` (HSB) and mines it's `0000` (RGB). This is something that needs to be addressed in #1 as for now it's only working with RGB. A PR would be welcomed! :D

**Comment by czebe** [comment]
Ahhha!! I didn't notice that. Loading the swatch in Photoshop shows the correct black/white colors.

**Comment by czebe** [comment]
Ok, I figured out how to change the color space for swatches in Photoshop. HSB swatches are exported by default. You need to click the RGB color mode on the color picker *before* creating the swatch. Exporting after this, works just as expected. This is somewhat counterintuitive, as designers would never switch to this mode in the color picker:

![color_picker](https://user-images.githubusercontent.com/2728671/31651696-f38a229c-b31c-11e7-9ad0-c7540059af4d.png)

**Comment by czebe** [comment]
Hi, I'm beginning to get my head around the ACO file format. One thing that I don't get yet, maybe you can help:
On line #44:
const getHex = (color) => color.slice(0, 2)

What's the rest of the color definition mean?
Eg. #010000 is represented by bytes: 0102 0000 0000. What the heck is 01*02*?
